jans-pycloudlib/jans/pycloudlib/persistence/sql.py

```
@@ -109,6 +148,67 @@ def connect_args(self):
+            RuntimeError: If Cloud SQL Connector library is not installed.
+
+        Example:
+            >>> adapter = PostgresqlAdapter()
+            >>> creator = adapter.get_cloudsql_connection_creator()
+            >>> engine = create_engine("postgresql+pg8000://", creator=creator)
+        """
+        if not CLOUDSQL_CONNECTOR_AVAILABLE:
+            raise RuntimeError(
+                "Cloud SQL Python Connector is not installed. "
+                "Install with: pip install 'jans-pycloudlib[cloudsql]'"
+            )
```

Ruff (0.14.6)

Avoid specifying long messages outside the exception class

(TRY003)